tools/Config.mk

```diff
@@ -523,12 +523,18 @@ ifeq ($(CONFIG_ARCH_COVERAGE),y)
 endif
 
 ifeq ($(CONFIG_WINDOWS_NATIVE),y)
+
+define NEWLINE
```

Comment:
why need append NEWLINE to the command?

Reply:
to dismiss an error, for example OBJS={A,B,C} , in the runing the statement $(foreach OBJ, $(OBJS), $(NEWLINE) $(call DELFILE,$(OBJ))) will be expanded as:
if exist A del /f /q A   **if** exist B del /f /q B    if exist C del /f /q C     there's an error:
there should no 'if'   ^
and the problem can not be solved neitherr  'space' nor ';'